src/molecule/dependency/base.py

```diff
@@ -80,13 +82,32 @@ def execute_with_retries(self):
         LOG.error(str(exception), self._sh_command)
         util.sysexit(getattr(exception, "exit_code", constants.RC_UNKNOWN_ERROR))
 
-    @abc.abstractmethod
-    def execute(self):  # pragma: no cover
+    def execute(self):
         """
         Execute ``cmd`` and returns None.
 
         :return: None
         """
+        if os.path.isfile("galaxy.yml"):
```

Comment:
This check is too naive to be effective. For example, we have our molecule scenarios stored under `tests/integration/molecule`, which means that we run Molecule from the `tests/integration` collection subfolder where galaxy.yml file will never be found.

Reply:
I will address this because molecule already looks for repository root location as is used in config file lookup, so we should be able to re-use repo-root path and avoid depending on cwd.